auth0_to_hubspot/program.py

```diff
@@ -0,0 +1,49 @@
+"""This program adds new Auth0 users to HubSpot as contacts."""
+
+import datetime
+import os
+
+from autokitteh.auth0 import auth0_client
+from autokitteh.hubspot import hubspot_client
+from hubspot.crm.contacts import SimplePublicObjectInput
+
+auth0 = auth0_client("auth0_conn")
+hubspot = hubspot_client("hubspot_conn")
+
+LOOKUP_HOURS = int(os.getenv("HOURS"))
```

Comment:
Constants should usually be above global variables, not below

Reply:
Done